mobile/src/helpers/navigation/get-screens-by-auth/get-screens-by-auth.helper.ts

```diff
@@ -0,0 +1,14 @@
+import { NavigationItem } from '~/common/types/types';
+
+const getScreensByAuth = (
+  screens: NavigationItem[],
+  hasUser: boolean,
+): NavigationItem[] => {
+  if (!hasUser) {
+    return screens.filter((screen) => screen.requireAuth === hasUser);
```

Comment:
I think it would be enough, and maybe `isAuthRequired` would be more suitable naming 🤔 
```suggestion
    return screens.filter((screen) => !screen.requireAuth);
```